lib/cisco_node_utils/evpn_stormcontrol.rb

```diff
@@ -55,16 +55,19 @@ def destroy
       config_stormcontrol
     end
 
-    def self.broadcast
-      config_get('evpn_stormcontrol', 'broadcast')
-    end
-
-    def self.multicast
-      config_get('evpn_stormcontrol', 'multicast')
+    # Create a hash of all stormcontrol instances
+    def self.stormcontrol
+      hash = {}
+      type_list = %w(broadcast multicast unicast)
+      type_list.each do |type|
+        level = config_get('evpn_stormcontrol', type)
+        hash[type] = EvpnStormcontrol.new(type, level, false) if level
```

Comment:
Based on the documentation on the puppet side, `level` is a property, not a parameter. If it's needed to uniquely identify the resource it should be a parameter for puppet.

Reply:
As discussed, the level is not required to uniquely identify the resource.